example_client_test.go

```diff
@@ -102,3 +103,47 @@ func ExampleMultiGetWithOptions() {
 	// Ranges: [{time-serie-1 map[] [{4 2}]} {time-serie-2 map[] [{4 10}]}]
 	// Ranges with labels: [{time-serie-1 map[az:us-east-1 machine:machine-1] [{4 2}]} {time-serie-2 map[az:us-east-1 machine:machine-2] [{4 10}]}]
 }
+
+// Exemplifies the usage of MultiAdd.
+func ExampleClient_MultiAdd() {
+	host := "localhost:6379"
+	password := ""
+	pool := &redis.Pool{Dial: func() (redis.Conn, error) {
+		return redis.Dial("tcp", host, redis.DialPassword(password))
+	}}
+	client := redistimeseries.NewClientFromPool(pool, "ts-client-1")
+
+	labels1 := map[string]string{
+		"machine": "machine-1",
+		"az":      "us-east-1",
+	}
+	labels2 := map[string]string{
+		"machine": "machine-2",
+		"az":      "us-east-1",
+	}
+
+	err := client.CreateKeyWithOptions("timeserie-1", redistimeseries.CreateOptions{RetentionMSecs: 0, Labels: labels1})
```

Comment:
@filipecosta90 `RetentionMSecs: 0` without comment might be confusing, I would have remove it